utils/mods.py

```diff
@@ -0,0 +1,186 @@
+        "crafted.stat_1671376347",  # Crafted lightning resist
+        "crafted.stat_4220027924",  # Crafted cold resist
+    }
+
+    dual_resist_ids = {
+        "explicit.stat_2915988346",  # Explicit fire and cold resists
+        "explicit.stat_3441501978",  # Explicit fire and lightning resists
+        "explicit.stat_4277795662",  # Explicit cold and lightning resists
+        "implicit.stat_2915988346",  # Implicit fire and cold resists
+        "implicit.stat_3441501978",  # Implicit fire and lightning resists
+        "implicit.stat_4277795662",  # Implicit cold and lightning resists
+        "crafted.stat_2915988346",  # Crafted fire and cold resists
+        "crafted.stat_3441501978",  # Crafted fire and lightning resists
+        "crafted.stat_4277795662",  # Crafted cold and lightning resists
+    }
+
+    triple_resist_ids = {
+        "explicit.stat_2901986750",  # Explicit all-res
+        "implicit.stat_2901986750",  # Implicit all-res
+        "crafted.stat_2901986750",  # Crafted all-res
+    }
+
+    solo_chaos_resist_ids = {
+        "explicit.stat_2923486259",  # Explicit chaos resist
+        "implicit.stat_2923486259",  # Implicit chaos resist
+    }
+
+    dual_chaos_resist_ids = {
+        "crafted.stat_378817135",  # Crafted fire and chaos resists
+        "crafted.stat_3393628375",  # Crafted cold and chaos resists
+        "crafted.stat_3465022881",  # Crafted lightning and chaos resists
+    }
+
+    life_ids = {
+        "explicit.stat_3299347043",  # Explicit maximum life
+        "implicit.stat_3299347043",  # Implicit maximum life
+    }
+
+    possible_ids = solo_resist_ids.copy()
+    possible_ids.update(dual_resist_ids)
+    possible_ids.update(triple_resist_ids)
+    possible_ids.update(solo_chaos_resist_ids)
+    possible_ids.update(dual_chaos_resist_ids)
+    possible_ids.update(life_ids)
+
+    combined_filters = []
+
+    for i in j["query"]["stats"][0]["filters"]:
+        if i["id"] in possible_ids:
+            if i["id"] in solo_resist_ids:
+                total_ele_resists += int(i["value"]["min"])
+                combined_filters.append(i)
+
+            elif i["id"] in dual_resist_ids:
+                total_ele_resists += 2 * int(i["value"]["min"])
+                combined_filters.append(i)
+
+            elif i["id"] in triple_resist_ids:
+                total_ele_resists += 3 * int(i["value"]["min"])
+                combined_filters.append(i)
+
+            elif i["id"] in solo_chaos_resist_ids:
+                total_chaos_resist += int(i["value"]["min"])
+                combined_filters.append(i)
+
+            elif i["id"] in dual_chaos_resist_ids:
+                total_chaos_resist += int(i["value"]["min"])
+                total_ele_resists += int(i["value"]["min"])
+                combined_filters.append(i)
+
+            # Maximum life
+            elif i["id"] in life_ids:
+                total_life += int(i["value"]["min"])
+                combined_filters.append(i)
+
```

Comment:
Can you add a elif check for strength as well and add strength/10 to total_life

Reply:
In a later PR, yes. However, the primary goal with this PR are the Item class/parsing/searching changes.

Reply:
Sure, just adding a note so it wont be forgotten